Podcast/Player.swift

```diff
@@ -199,12 +199,25 @@ class Player: NSObject {
     func setProgress(progress: Double) {
         if let duration = player.currentItem?.duration {
             if !duration.isIndefinite {
-                player.currentItem!.seek(to: CMTime(seconds: duration.seconds * min(max(progress, 0.0), 1.0), preferredTimescale: CMTimeScale(1.0)), completionHandler: nil)
-                delegate?.updateUIForPlayback()
+                player.currentItem!.seek(to: CMTime(seconds: duration.seconds * min(max(progress, 0.0), 1.0), preferredTimescale: CMTimeScale(1.0)), completionHandler: { (_) in
+                    self.isScrubbing = false
+                    self.delegate?.updateUIForPlayback()
+                    self.updateNowPlayingInfo()
```

Comment:
also can you make a spec comment for updateNowPlayingInfo()

Reply:
Sure thing.